rustc-ice-2024-11-04T12_36_50-74186.txt

```diff
@@ -0,0 +1,63 @@
+thread 'rustc' panicked at /rustc/3f1be1ec7ec3d8e80beb381ee82164a0aa3ca777/compiler/rustc_type_ir/src/binder.rs:777:9:
```

Comment:
Not sure we need this file :D

Reply:
Yep I meant to ask about this, not sure where it came from or if it was used elsewhere when running `cargo test` or the like but I'll get rid of it.